updating/updating_a_cluster/eus-eus-update.adoc

```diff
@@ -53,6 +53,7 @@ include::modules/updating-eus-to-eus-upgrade-cli.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
+* xref:../../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
```

Comment:
I'm really sorry, I think I gave you a slightly incorrect filepath in my review of your other PR, here's what it should be:
```suggestion
* xref:../../updating/updating_a_cluster/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
```